test/e2e/network_segmentation.go

```diff
@@ -1808,6 +1809,47 @@ spec:
 			),
 		)
 	})
+
+	Context("UDN with limited host subnet", func() {
+		It("check ovnkube-node pod restart when node is not assigned with UDN host subnet", func() {
+			nodes, err := e2enode.GetReadySchedulableNodes(context.TODO(), f.ClientSet)
+			framework.ExpectNoError(err)
+			if len(nodes.Items) < 3 {
+				ginkgo.Skip("requires at least 3 Nodes")
+			}
+
+			By("creating the network with limited host subnet configuration")
+			netConfig := networkAttachmentConfigParams{
+				name:     nadName,
+				topology: "layer3",
+				cidr:     correctCIDRFamily(cs, "192.168.100.0/25/26", "2014:100:200::/63/64"),
```

Comment:
Can you add a brief comment with the reasoning behind your e2e test? Her I understand that you're assigning a pod subnet of /25 to the cluster and of /26 to each node. So there will be only two /26 subnets available and the third node will be left without a subnet. The same goes for IPv6. Is this correct?

Reply:
yes @ricky-rav , you are right, added comment to reflect this.